bugbot/rules/variant_expiration.py

```diff
@@ -296,6 +296,8 @@ def handle_bug(self, bug, data):
             "expiration": bug_expiration.strftime("%Y-%m-%d"),
         }
 
+        needinfo_flag_ids = self.get_needinfo_ids(bug)
+
```

Comment:
We could iterate over the method's results directly without assigning them to a variable. This would avoid generating the result in cases where it is not needed and would be more compact.